src/wamr/CMakeLists.txt

```diff
@@ -11,6 +11,7 @@ set(WAMR_BUILD_SPEC_TEST 0)
 add_definitions(-DWAMR_FAASM=1)
 
 # Set AOT mode and JIT for code generation
+set(WAMR_BUILD_INTERPRETER 1)
```

Comment:
This was set by default implicitly through setting `WAMR_BUILD_AOT`, but I prefer to make it explicit.